pkg/liqoctl/install/aks/provider.go

```diff
@@ -78,27 +77,23 @@ func (k *aksProvider) ValidateCommandArguments(flags *flag.FlagSet) (err error)
 	if err != nil {
 		return err
 	}
-	klog.V(3).Infof("AKS SubscriptionID: %v", k.subscriptionID)
```

Comment:
I would personally keep the possibility to show debug information in case the `--debug` flag is set (could use an helper function, to make it unrelated from klog).